frontend/src/components/CcfriEstimator.vue

```diff
@@ -175,7 +175,7 @@
                       <v-icon class="pt-1" size="small" color="white"> mdi-help </v-icon>
                     </v-card>
                   </template>
-                  <div>{{ careTypeToolTip }}</div>
+                  <div v-html="careTypeToolTip"></div>
```

Comment:
Looking at the existing code, I'm thinking we should ditch the computed() methods altogether and just put the content in directly. They don't get more complex than an v-if/v-else statement and having them as computed just adds unnecessary complexity.